report/report.go

```diff
@@ -94,6 +95,7 @@ func (g Generator) reportObjFromDependency(dep models.Dependency) (*models.Repor
 		}
 	default:
 		log.Println("Unable to determine repo source for ", dep.Name)
+		return nil, fmt.Errorf("unable to determine repo source for %s, must add to map", dep.Name)
```

Comment:
`must add to map` is somewhat cryptic. I assume you are referring to the switch statement above, correct? If so, you could just swap `add to map` for something like `add source to generator` or just leave out the second part of the error message. I assume this will be wrapped by the caller as well.